tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -330,14 +330,11 @@ public ImmutableSet<Integer> getVoteCountMenu(Organization ignoredOrg) {
                             PERMANENT_VOTES);
         }
 
-        // The following methods were moved here from UserRegistry
-        // TODO: remove this todo notice
```

Comment:
?!

Reply:
It should have been a review comment. Apologies.

Reply:
no problem! It was just puzzling and funny